tests/test_proxy_connect.py

```diff
@@ -100,9 +117,10 @@ def test_https_tunnel_without_leak_proxy_authorization_header(self):
         with LogCapture() as l:
             yield crawler.crawl(seed=request)
         self._assert_got_response_code(200, l)
-        echo = json.loads(crawler.spider.meta['responses'][0].body)
+        echo = json.loads(crawler.spider.meta['responses'][0].body.decode('utf-8'))
```

Comment:
Is this a Scrapy `Response` object? Because then:

```suggestion
        echo = json.loads(crawler.spider.meta['responses'][0].text)
```